homeassistant/components/recorder/history.py

```diff
@@ -110,52 +109,48 @@
     StateAttributes.shared_attrs,
 ]
 
-HISTORY_BAKERY = "recorder_history_bakery"
 
+def _schema_version(hass: HomeAssistant) -> int:
+    return recorder.get_instance(hass).schema_version
 
-def bake_query_and_join_attributes(
-    hass: HomeAssistant, no_attributes: bool, include_last_changed: bool = True
+
+def lambda_stmt_and_join_attributes(
+    schema_version: int, no_attributes: bool, include_last_changed: bool = True
 ) -> tuple[Any, bool]:
```

Comment:
https://docs.sqlalchemy.org/en/14/core/sqlelement.html#sqlalchemy.sql.expression.lambda_stmt

> The object returned is an instance of [StatementLambdaElement](https://docs.sqlalchemy.org/en/14/core/foundation.html#sqlalchemy.sql.expression.StatementLambdaElement).
```suggestion
) -> tuple[StatementLambdaElement, bool]:
```